File: python/7 kyu/161.py
# https://www.codewars.com/kata/5a0366f12b651dbfa300000c/train/python

# Passed

def arbitrate(inp, n):
    if "1" not in inp:
        # no master is requesting access
        result = "0" * n
        return result
    
    idx = inp.index("1")
    result = ["0"] * n
    result[idx] = "1"
    result = "".join(result)
    
    return result

output = arbitrate("001000101", 9)
print(output)

File: nodejs/7 kyu/161.js
// https://www.codewars.com/kata/5502c9e7b3216ec63c0001aa/train/javascript

// Passed

function openOrSenior(data) {
  let memberPositions = data.map((memberData) =>
    memberData[0] >= 55 && memberData[1] > 7 ? "Senior" : "Open"
  );
  return memberPositions;
}

const output = openOrSenior([
  [45, 12],
  [55, 21],
  [19, -2],
  [104, 20],
]);
console.log(output);